qemu/tests/cfg/timerdevice.cfg

```diff
@@ -58,6 +58,10 @@
                 - base_localtime:
                     only Windows
                     rtc_base = localtime
+                - base_date:
+                    Windows:
+                        only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
+                    rtc_base = "2006-06-17"
```

Comment:
For windows guest, now we only test future time, such as "2020-05-30"

Reply:
Yes, it should be, as I talked in our weekly meeting.